tasks/azure.py

```diff
@@ -0,0 +1,145 @@
+from ast import literal_eval
+from invoke import task
+from subprocess import check_output, run
+
+
+RESOURCE_GROUP = "faasm"
+LOCATION = "eastus"
```

Comment:
Should we run in Europe? If the client machine is in the same region I guess it's only the latency between us and the client that will be noticeable.